packages/build-utils/test/unit.builds-and-routes-detector.test.ts

```diff
@@ -2231,35 +2231,63 @@ describe('Test `detectBuilders` with `featHandleMiss=true`', () => {
 
   it('no package.json + no build + root-level "middleware.js"', async () => {
     const files = ['middleware.js', 'index.html', 'web/middleware.js'];
-    const { builders, errors } = await detectBuilders(files, null, {
-      featHandleMiss,
-    });
-    expect(builders![0].use).toBe('@vercel/node');
-    expect(builders![0].src).toBe('middleware.js');
+    const { builders, middlewareRoutes, errors } = await detectBuilders(
```

Comment:
Lets also assert any other routes are what we expect here

Reply:
0bec2f2dc3efd1346dec9c453af402e718f658b4